### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,80 @@
...
+    private int passengerCapacity;
+    private int yearOfManufacturing;
 
+    private Plane(PlaneBuilder builder) {
+        this.manufacturer = builder.manufacturer;
+        this.model = builder.model;
+        this.boardNumber = builder.boardNumber;
+        this.passengerCapacity = builder.passengerCapacity;
+        this.yearOfManufacturing = builder.yearOfManufacturing;
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: what's the purpose of using `this` here?

also, please remove the blank line

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,80 @@
...
+    public static class PlaneBuilder {
+        private String manufacturer;
+        private String model;
+        private int boardNumber;
+        private int passengerCapacity;
+        private int yearOfManufacturing;
+
+        public PlaneBuilder setManufacturer(String manufacturer) {
+            this.manufacturer = manufacturer;
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: please remove this blank line in all setters

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/Plane.java]: done

File: src/main/java/core/basesyntax/Plane.java (hunk 1 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/Plane.java]: got it.
fixed

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**